js/src/controls/WidgetControl.js

```diff
@@ -0,0 +1,94 @@
+var widgets = require('@jupyter-widgets/base');
+var _ = require('underscore');
+var L = require('../leaflet.js');
+var control = require('./Control.js');
+var LeafletControlView = control.LeafletControlView;
+var LeafletControlModel = control.LeafletControlModel;
+
+L.Control.WidgetControl = L.Control.extend({
+    options: {
+        position: 'topleft',
+        widget : 'null',
```

Comment:
As far as I can see you never use this `widget` option, I guess you can remove it